test/lib/ansible_test/_data/sanity/yamllint/yamllinter.py

```diff
@@ -119,7 +175,6 @@ def get_module_docs(self, path, contents):
         """
         module_doc_types = [
             'DOCUMENTATION',
-            'EXAMPLES',
```

Comment:
Even though `ansible-doc` does not require valid YAML for examples, we've been requiring it since Ansible 2.5. I'm not sure this is something we want to change.

We've tried to keep examples as valid YAML because users often copy them and expect them to work. If there's interest in relaxing this I'd bring it up with the docs team and perhaps discuss at the next core IRC meeting.

Reply:
Currently, though, there's an awful lot which are not valid yaml....

Reply:
@abadger The examples with invalid yaml are only detected by the new test, correct? I don't see any ignores for the current yamllint test.

Reply:
Is there a way to add yamllint to the ignore.txt?  I can add the files to that as an alternative to excluding examples.

Reply:
Correct, the invalid yaml is found with the new test.

Reply:
Yes, almost all of the sanity tests support ignores, including the yamllint test.

An ignore for yamllint would look like:

```
path/to/file.py yamllint:errorcode
```

Reply:
Any hints on choosing an appropriate errorcode and where that would need to be registered?

Reply:
Alternately, I was unable to get ansible-test to ignore the yamllint test for certain files without an error code using:
```
test/integration/targets/exos_facts/tasks/main.yaml yamllint      
```

Is my format for ignoring tests which have no errorcode wrong?

Reply:
An error code is required.

The code added in this PR for parse errors is `unparsable-with-libyaml`, so using your example you'd want:

```
test/integration/targets/exos_facts/tasks/main.yaml yamllint:unparsable-with-libyaml
```

Reply:
Doh, and I even wrote that and then forgot it... Must be getting old.  I only looked in yamllint, not in yamllinter.

Okay, all the non-yaml examples are now ignored on a module by module basis.